tests/elements-regression/src/pages/base-page.js

```diff
@@ -4,14 +4,21 @@ module.exports = class BasePage {
 	 */
 	page;
 
+	/**
+	 * @type {{}}
+	 */
+	config = {};
+
 	/**
 	 * @param {import('@playwright/test').Page} page
+	 * @param {{}}                              config
 	 */
-	constructor( page ) {
+	constructor( page, config ) {
```

Comment:
If the config is only holds the page id maybe put it into the editor page only?

Reply:
It's more flexible than override the constructor on the extended class. 
Overriding will make it harder to change the base in the future. 
What do you think?

Reply:
Done. I added a `setPageId` method.